src/grid/DomainGridView.php

```diff
@@ -162,9 +162,12 @@ public function columns()
                     'placement' => 'bottom',
                     'selector' => 'span',
                 ],
-                'pluginOptions' => [
-                    'offColor' => 'warning',
-                ],
+                'pluginOptions' => function($model) {
+                    return [
+                        'offColor' => 'warning',
+                        'readonly' => !(bool)$model->isWPChangeable(),
```

Comment:
```suggestion
                        'readonly' => !$model->isWPChangeable(),
```

Возвращаемый тип – строго буль, приводить не нужно